ironfish-rust/src/transaction/tests.rs

```diff
@@ -71,7 +71,7 @@ fn test_transaction() {
         spender_key.public_address(),
     );
 
-    let mut transaction = ProposedTransaction::new(spender_key);
+    let mut transaction = ProposedTransaction::new(spender_key, TransactionVersion::V2);
```

Comment:
I'm wondering if it would be beneficial for this kind of tests to have a `TransactionVersion::latest()` accessor? When we introduce V3 transactions, I *think* we will want these tests to use V3.

We could also parametrize these tests (making them run for all possible versions), but this would be out of scope for this PR.

Reply:
Great idea - added